services/graphql-server/src/graphql/definitions/platform/content/index.js

```diff
@@ -275,6 +275,7 @@ input ContentSitemapNewsUrlsQueryInput {
 
 input AllPublishedContentQueryInput {
   siteId: ObjectID
+  after: Date
```

Comment:
You'll also need to add this to `PublishedContentCountsQueryInput` since you're using the `after` value in that resolver.